pcdet/models/dense_heads/anchor_head_template.py

```diff
@@ -101,14 +101,24 @@ def assign_targets(self, gt_boxes):
     def get_cls_layer_loss(self):
         cls_preds = self.forward_ret_dict['cls_preds']
         box_cls_labels = self.forward_ret_dict['box_cls_labels']
+        cls_loss = self.get_cls_layer_loss(
+            cls_preds=cls_preds, box_cls_labels=box_cls_labels, num_class=self.num_class
+
+        )
```

Comment:
This function is calling itself if I'm not mistaken. How could that work?

Maybe you wanted to call `get_cls_layer_loss_one_set`?

Reply:
Oh damn good catch